src/Parser.js

```diff
@@ -678,7 +618,7 @@ export default class Parser {
      */
     parseColorGroup(optional: boolean): ?ParseNode<"color-token"> {
         const res = this.parseStringGroup("color", optional);
-        if (!res) {
+        if (res == null) {
```

Comment:
This doesn't change the behaviour does it?

Reply:
@kevinbarabash It doesn't.